jenkins/Jenkinsfile.build

```diff
@@ -241,7 +238,8 @@ pipeline {
                     string(name: "OS_FILTER", value: params.OS_FILTER),
                     string(name: "ARCH_FILTER", value: params.ARCH_FILTER),
                     booleanParam(name: "DAILY", value: params.DAILY),
-                    booleanParam(name: "PUBLISH", value: params.PUBLISH)
+                    booleanParam(name: "PUBLISH", value: params.PUBLISH),
+                    booleanParam(name: "FORCE_REBUILD", value: params.FORCE_REBUILD)
```

Comment:
I suspect you're focusing on Linux at the moment -- just a note to add this `FORCE_REBUILD` parameter to the MacOS and Windows stages as well!